src/components/IssuesPage.jsx

```diff
@@ -1,12 +1,41 @@
 import React, { useState } from 'react';
 import IssuesList from './IssuesList';
 const IssuesPage = (props) => {
+  const example = [
+    {
+      description: 'test',
+      id: 'test',
+      reported_count: 'test',
+      task_owner: 'test',
+      username: 'test',
+      user_id: 'test',
+      upvoted_count: 'test',
+      resolution_status: 'test',
+      confirm_resolved_count: 'test',
+      date_issued: 'test',
+      date_marked_in_progress: 'test',
+      date_marked_resolved: 'test',
+    },
+    {
+      description: 'test2',
+      id: 'test2',
+      reported_count: 'test2',
+      task_owner: 'test2',
+      username: 'test2',
+      user_id: 'test2',
+      upvoted_count: 'test2',
+      resolution_status: 'test2',
+      confirm_resolved_count: 'test2',
+      date_issued: 'test2',
+      date_marked_in_progress: 'test2',
+      date_marked_resolved: 'test2',
+    },
+  ];
+
   return (
     <div>
       {/* <Statuses /> */}
-      <IssuesList
-      // issues={props.}
-      />
+      <IssuesList issues={example} />
     </div>
```

Comment:
Consider moving example outside the scope of IssuesPage and then making your `() => {};` function a `() => ();` function due to instant return.

Reply:
fixed requested changes

Reply:
Thanks for adjusting!